updates/sync.js

```diff
@@ -0,0 +1,110 @@
+var _ = require('underscore'),
+	rp = require('request-promise');
+	keystone = require('keystone');
+	User = keystone.list('User');
+
+module.exports = {
+
+	//TODO Continuous sync
+	syncUsersToDb: function () {
+		var options = {
+			uri: 'https://slack.com/api/users.list?',
+			qs: {
+				token: 'xoxp-2186825377-14603377189-42848621732-2e85220052'
+			},
+			json: true
+		};
+		
+		var admin = { 'name.first': 'Admin', 'name.last': 'User', 'email': 'development@icapps.com', 'password': 'admin', 'isAdmin': true };
+		
+		
+		_removeTable('User').then(function(){
+			_createAdmin(admin);
+			_getInfoFromSlack(options);
+		});
+
+	}
+
+
+};
+
+function _getInfoFromSlack(options) {
+
+	var userArray = [];
+
+	rp(options)
+		.then(function (res) {
+			_.each(res.members, function (member) {
```

Comment:
Always use _.forEach() to loop an array. _.each and _.forEach have different logic